handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
+
+## One-time expenses
+**Laptop (engineers):** $3500
+
+**Laptop (non-engineers):** $2000
+
+**Laptop coverage:** $380
+
+**Monitor:** $380
+
+**Headphones:** $200
+
+**Keyboard:** $130
+
+**Mouse or trackpad:** $80 or $130
+
+**USB-C Adapter:** $70
+
+**HDMI/monitor cable:** $15
+
+**Laptop stand:** $40
+
+**Desk:** $500
+
+**Ergonomic chair:** $200
+
+
+
+## Ongoing expenses
+**Software:** case by case, please get your manager’s approval.
+
+**Internet/WiFi plan:** $60/month
+
+**Cell phone plan:** $75/month (no need to expense, this is automatically added onto your pay).
+
+**Coworking/office space:** TBD
+
+**Professional development (conferences, courses):** $1000/year
+
+**Moving (moving, packing, shipping, etc.):** $1000/year
+
+**Wellness:** $85/month
+
+
+
+## Other expenses:
+
+**Laptop repairs:** if your laptop requires repairs, we cover up to $500 a year on repairs. If necessary repairs exceed the yearly budget please reach out to PeopleOps for approval.
+
+**Laptop upgrade:** if your laptop is 4 years or older, you are eligible for a new laptop (same budget as new laptop applies).
+
+**Travel expenses:** please see our page on travel budgets. 
+
+
+## How to submit Expenses:
```

Comment:
This section should just be a link to the page I added a while back! https://about.sourcegraph.com/handbook/people-ops/expenses